en/docs/administer/logging-and-monitoring/logging/setting-up-logging.md

```diff
@@ -38,7 +38,34 @@ Following is the default configuration for carbon logs and the default values ca
      ```
    
    The log growth of carbon logs can be managed by the configurations discussed in [Managing log growth]({{base_path}}/administer/product-administration/monitoring/logging/managing-log-growth) guide.
-   
+
+###Enable Logs for a Tenant
+
+By default, logs of all the tenants are written to a single log file. You can configure to write logs of tenants to a different log file using the below configurations to the log4j2.properties file.
```

Comment:
Shall we change it to the following?

**In order to write logs of tenants to a different log file the following configurations can be added to the log4j2.properties file.**

Reply:
fixed